examples/units_and_coordinates/map_slit_extraction.py

```diff
@@ -0,0 +1,61 @@
+"""
+=====================================================
+Extracting intensity of a map between two coordinates
+=====================================================
+
+In this example we will define a slit in world coordinates and then extract the intensity values of all the pixels that the path intersects with.
+"""
+import matplotlib.pyplot as plt
+
+import astropy.units as u
+from astropy.coordinates import SkyCoord
+
+import sunpy.map
+from sunpy.data.sample import AIA_171_ROLL_IMAGE
+
+###############################################################################
+# First we construct a map, using some sample data.
+aia_map = sunpy.map.Map(AIA_171_ROLL_IMAGE)
+
+
+###############################################################################
+# Next we define a path in a `~astropy.coordinates.SkyCoord` object.
+# In this example we are just going to use a straight line, however a path with
+# any number of points can be used, and the slit will be defined as straight
+# line segments between all the points.
+line_limits = SkyCoord([-1024, -908], [20, 633], unit=(u.arcsec, u.arcsec),
```

Comment:
```suggestion
line_coords = SkyCoord([-1024, -908], [20, 633], unit=(u.arcsec, u.arcsec),
```
I don't think `limits` is the right word here?